index.html

```diff
@@ -31,7 +31,7 @@ <h1>Reading Tracker</h1>
         <p>
           There are <span id="daysLeftDisplay"></span> days left in the year.
         </p>
-        <p>
+        <p id="frequencyDisplayText">
```

Comment:
I added an id to this tag, so that it can be accessed in the JS code to write the message and also reset the tag